manage/deploying/front-end/nginx.rst

```diff
@@ -140,11 +148,13 @@ Deployment configuration
 ========================
 
 `gocept.nginx <https://pypi.python.org/pypi/gocept.nginx/>`_ supports a special deployment configuration where you manually configure all directories.
-One important reason why you might wish to do this, is to change the location of the ``pid`` file.
+One important reason why you might wish to do this is to change the location of the ``pid`` file.
 Normally this file would be created in ``parts``, which is deleted and recreated when you re-run buildout.
 This interferes with reliably restarting nginx, since the pid file may have been deleted since startup. In this case, you need to manually kill nginx to get things back on track.
 
-Example deployment configuration in ``production.cfg``::
+Example deployment configuration in ``production.cfg``:
+
+.. code-block:: nginx
 
```

Comment:
This needs testing first, I am not sure that we have code highlight for nginx, and if we have that that the colors are OK, we never used that or tested that.